### ISSUE TITLE
Pushing to a git repo sync requires unlocking FIDO2 backed ssh key twice

### ISSUE THREAD

**Issue by slrtbtfs** [microsoft/vscode#145822]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.65.2
- OS Version: Fedora 36

Steps to Reproduce:

Reproducing this requires owning a YubiKey or similar device

1. Setup a FIDO2 backed ssh key [as described on the Yubicos website](https://developers.yubico.com/SSH/Securing_SSH_with_FIDO2.html ). 
2. Add that key to a remote git host, like GitHub or a GitLab Instance.
4. Open a local clone of a git repo hosted there.
5. Create a commit.
6. Click the "Sync Changes" Button in the git view.
7. The YubiKey lights up and needs to be touched to unlock the ssh key
8. After that the YubiKey lights up again and needs to be pressed *again*.

Ideally, touching the YubiKey would be required only once.

Note that the password part of unlocking the ssh key is handled by gnome-keyring. I suspect otherwise I'd hit microsoft/vscode-remote-release#6486.

**Comment by yume-chan** [comment]
Sync does pull + push, so maybe that's why?

**Comment by slrtbtfs** [comment]
That's my guess as well.

Still, it'd be nice if there was a way to sync with only unlocking the key once.

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!
